cmd/context/utils.go

```diff
@@ -222,9 +222,7 @@ func LoadContextFromPath(ctx context.Context, namespace, k8sContext, path string
 	}
 	ctxResource, err := model.GetContextResource(path)
 	if err != nil {
-		if !errors.Is(err, oktetoErrors.ErrManifestNotFound) {
-			return err
-		}
+		return err
```

Comment:
I think the logic within this function was not right. I'll try to explain myself. According to the comment, this function `initializes the okteto context taking into account command flags and manifest namespace/context fields`.

But, if a manifest were not found, we were throwing a panic because we were trying to set the namespace and context for a nil object.

Now, with this change, we are not considering never the namespace and context specified in the flag commands (the one we receive as parameter) if the manifest does not exist, which I think it is not correct.

I think we should create a context resource with the values from the flag, then, retrieve the context from the manifest and merge them.

Reply:
> Now, with this change, we are not considering never the namespace and context specified in the flag commands (the one we receive as parameter) if the manifest does not exist, which I think it is not correct.

i think this function is just for context from manifest files, and additionaly (which i think should not be part of this func) we override with flag values. we do consider the flag values when error, now that we added the new context command after the error. when file and flags values will be correct, when err because no file, flags will be taken in consideration also